src/resources/lang/de/wallet.php

```diff
@@ -0,0 +1,155 @@
+<?php
```

Comment:
I think you got more variant than @ppfeufer here - so probably use it instead the current version.

Reply:
The lines that I translated there are actually from the german EVE Client. That are the translations used by CCP as far as I could gather from my corp guys who have been willing to switchb their client to german.

The translations in this file sound an aweful lot like Google Translate to me.

Isn't there a ways to get the translations for those strings from CCP or even ESI with a language tag? I would be surprised if that's not available somewhere.